e2e_playwright/st_expander_state_test.py

```diff
@@ -14,17 +14,22 @@
 
 from playwright.sync_api import Page, expect
 
+from e2e_playwright.conftest import wait_for_app_run
+
 
 def test_expandable_state(app: Page):
     """Test whether expander state is not retained for a distinct expander."""
-    app.locator(".stButton button").nth(0).click()
-    app.locator("[data-testid='stExpander'] summary").click()
+    app.get_by_test_id("stButton").nth(0).locator("button").click()
+    wait_for_app_run(app, wait_delay=500)
+    app.get_by_test_id("stExpander").locator("summary").click()
+    expect(app.get_by_test_id("stExpanderDetails")).to_be_visible()
```

Comment:
nit: Can we just get rid of this because line 26 will do the same thing?

Reply:
Yep, indeed. We can remove this line.